easybuild/easyconfigs/f/FFmpeg/FFmpeg-4.1.3-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'ConfigureMake'
+
+name = 'FFmpeg'
+version = '4.1.3'
+
+homepage = 'https://www.ffmpeg.org/'
+description = """A complete, cross-platform solution to record, convert and stream audio and video."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
```

Comment:
@Micket Don't we pay a price performance-wise by not compiling `FFmpeg` with Intel compilers?

Reply:
@boegel Not really. x264/x265 were already at GCCcore level. And for all the builtin stuff FFmpeg does a lot of manual assembly code in all the performance hotspots.
Secondly, I somehow doubt HPC time spent encoding movies are going to be insignificant. 
Thirdly, i wouldn't think that icc is necessarily really that amazing at compiling software like this.

But, just to be sure, I actually just tested building FFmpeg and FFmpeg+x264 at iccifort level, and benchmarked converting and reencoding some full HD movie trailer i grabbed online. iccifort versions were slightly worse in all tests i made (by ~5%)

Reply:
Excellent, thanks for diving into this!